web/client/translations/data.en-US.json

```diff
@@ -104,6 +104,21 @@
             "templateFormatInfoAlert2": "Use ${ attribute } to wrap the properties you need to display",
             "templateFormatInfoAlertExample": "The id of the feature is ${ properties }",
             "templatePreview": "Template preview",
+            "visibilityLimitsTitle": "Visibility limits",
```

Comment:
probably these could have been an object 
"visibilityLimits": {
            "title": "Visibility limits",

but is fine as you did too, feels free to keep them as is

Reply:
I'm keeping current structure

Reply:
@MV88 I'll follow your suggestion, please ignore my previous comment